### DISCUSSION TITLE
Adobe Harman AIR on HMDI connected 4k TV box device

### DISCUSSION THREAD

**Discussion by as3wizard** [airsdk/Adobe-Runtime-Support#1640] [Q&A]
Has anyone ever successfully installed an AIR app onto any of these 4K TV box devices?

T95 pendoo wechip turewell
X88 X98 Ultra HD Android 11.0
NVIDIA SHIELD Android TV Pro
Amazon Fire TV stick
Google Chromecast Ultra
etc.

**Comment by lwmirkk** [comment]
We use several TV Box with success. Amazon Fire is one of them, x88 and x98 and all other similar SoC are OK too. *
Chromecast never tested it. I think it won`t work on it because chromecast is only a receiver (screen mirror and other things), not a complete OS like other TV Box with Android TV.

* Dependending which TV Box you use, you will have good or bad performance...
* Fire Stick 4k, Xiaomi Mi Box, Realme Stick and some others are the best cost benefit I think. NVIDIA is good too, but the cost is high.
